Tasks/AzureNLBManagement/task.json

```diff
@@ -0,0 +1,120 @@
+{
+	"id": "e94f1750-a6a8-11e6-be69-bdf37a7b15d8",
+	"name": "AzureNLBManagement",
+	"friendlyName": "Azure Network Load Balancer (Preview)",
+	"description": "Connect/Disconnect an Azure virtual machine's network interface to a Load Balancer's backend address pool",
+	"author": "Microsoft Corporation",
+	"helpMarkDown": "Replace with markdown to show in help",
+	"category": "Utility",
+	"visibility": [
+		"Build",
+		"Release", 
+        "Preview"
+    ],
+    "runsOn": [
+        "MachineGroup"
+    ],
+	"version": {
+		"Major": "1",
+		"Minor": "0",
+		"Patch": "0"
+	},
+	"minimumAgentVersion": "1.95.0",
+	"instanceNameFormat": "Azure Network Load Balancer: $(Action) VMs from $(LoadBalancer)",
```

Comment:
Check with PM on the resource here. This should be meaningful for both connect and disconnect actions.